serial_communication/BipedTeensyLCM/fix_std.hpp

```diff
@@ -0,0 +1,24 @@
+/* These function definitions are necessary for 
+   std to work on the Teensy microcontrollers
+ */
+
+#ifndef SLAVE_FIX_STD_HPP_
+#define SLAVE_FIX_STD_HPP_
+
+
+namespace std {
+  inline void __throw_bad_alloc()
+  {
+    Serial.println("Unable to allocate memory");
+    while(1);
```

Comment:
Yeah I wrote that, it was just copied over and shouldn't be in this commit. Yeah it's correct, the function is [[noreturn]] which means the compiler complains if it is capable of returning.